samples/boards/reel_board/mesh_badge/src/mesh.c

```diff
@@ -569,14 +609,20 @@ static void start_mesh(struct k_work *work)
 	int err;
 
 	err = provision_and_configure();
+
 	if (err < 0) {
-		board_show_text("Starting Mesh Failed", false,
-				K_SECONDS(2));
+		show_mesh_status(false, 0);
+	}
+}
+
+void show_mesh_status(bool mesh_started_ok, int addr)
```

Comment:
Could you please move this function up to its forward declaration? We try to avoid to avoid forward declaration unless it is impossible to avoid it.

And could you convert this function to static because it is only used in this file?